spec/controllers/ctc/questions/bank_account_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+describe Ctc::Questions::BankAccountController do
+  let(:intake) { create :ctc_intake, refund_payment_method: :direct_deposit, bank_account: (create :empty_bank_account) }
+
+  before do
+    sign_in intake.client
+  end
+
+  describe "#update" do
+    context "with no answer" do
+      let(:params) do
+        {}
+      end
+
+      it "re-renders the form with errors" do
+        post :update, params: params
```

Comment:
`post :update, params: {}` could we just do this and remove the `let(:params)` for this context